yal/main.py

```diff
@@ -128,8 +128,47 @@ async def process_message(self, message):
             message.session_event = Message.SESSION_EVENT.RESUME
             self.state_name = feedback_state
 
+        if self.user.metadata.get("is_tester") == "TRUE":
```

Comment:
should there be a default here so that if "is_tester" isn't in the metadata this doesn't throw an error?

Reply:
Yeah the default here is `None`
```
>>> metadata = {}
>>> print(metadata.get("is_tester"))
None
```